recipes/mathfu/all/conanfile.py

```diff
@@ -1,35 +1,43 @@
-from conans import ConanFile, tools
+from conan import ConanFile
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
 import os
 
+required_conan_version = ">=1.52.0"
+
 
 class MathfuConan(ConanFile):
     name = "mathfu"
     description = "C++ math library developed primarily for games focused on simplicity and efficiency."
-    topics = ("conan", "mathfu", "math", "geometry")
+    topics = ("math", "geometry")
     license = "Apache-2.0"
     homepage = "https://github.com/google/mathfu"
     url = "https://github.com/conan-io/conan-center-index"
-    settings = "os"
+    settings = "os", "arch", "compiler", "build_type"
     no_copy_source = True
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    def layout(self):
+        basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("vectorial/cci.20190628")
+        self.requires("vectorial/cci.20190628", transitive_headers=True, transitive_libs=True)
```

Comment:
`transitive_libs=True` is okay! 👍 

https://github.com/conan-io/conan/pull/12380/files#diff-ec95a95e490859a209a30e6970444d6f6184b2fd99b5930bc9dda14dafe80d25R275